xwiki-platform-core/xwiki-platform-uiextension/xwiki-platform-uiextension-api/src/main/java/org/xwiki/uiextension/internal/WikiUIExtensionParameters.java

```diff
@@ -133,11 +133,12 @@ public WikiUIExtensionParameters(String id, String rawParameters, ComponentManag
             properties.load(new StringReader(rawParameters));
             result = new HashMap<>();
             for (String key : properties.stringPropertyNames()) {
-                if (!Objects.equals(key, "")) {
+                if (!StringUtils.isBlank(key)) {
                     result.put(key, properties.getProperty(key));
                 }
             }
         } catch (IOException e) {
+            LOGGER.error("UIX parameters could not be read", e);
```

Comment:
A good practice is to always provide the most information possible in logging. For example here you should pass `rawParameters` IMO. See https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/JavaCodeStyle/#HLoggingBestPractices